R/slice.R

```diff
@@ -125,12 +125,19 @@ vec_assign_fallback <- function(x, i, value) {
   x
 }
 
-#' Create an index
+#' Create an index vector or a position
 #'
-#' This provides a means of standardizing common indexing methods such as
-#' integer, character or logical indexing. The input is a vector of one
-#' of these three types, and the output is always an integer vector that can
-#' be used for subsetting.
+#' These helpers provide a means of standardizing common indexing
+#' methods such as integer, character or logical indexing.
+#'
+#' * `vec_as_index()` accepts integer, character, or logical vectors
+#'   of any sizes. The output is always an integer vector of the same
```

Comment:
```suggestion
#'   of any size. The output is always an integer vector of the same
```

Reply:
The output isn't always the same size?

Reply:
oh of course

Reply:
This is now:

```
#'   [...] The output is always an integer vector that is
#'   suitable for subsetting with `[` or [vec_slice()]. It might be a
#'   different size than the input because negative selections are
#'   transformed to positive ones and logical vectors are transformed
#'   to a vector of indices for the `TRUE` positions.
```